### ISSUE TITLE
HuggingFace Serverless Inference API

### ISSUE THREAD

**Issue by debjan** [continuedev/continue#2122]
### Validations

- [X] I believe this is a way to improve. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement) that requests the same enhancement

### Problem

Hi, can HuggingFace Inference be set with "Serverless Inference API" instead "Inference Endpoint"?

**Comment by sestinj** [comment]
I agree, this would be nice! Linking to the docs [here](https://huggingface.co/docs/api-inference/en/index) in case anyone is interested in contributing. Not otherwise a top priority at least at the very moment

**Comment by vscodesyncsettings** [comment]
Please work on this :-)

**Comment by KertLynx** [comment]
I hope someone is working on this ...
